src/components/school-onboarding/onboardingForm.tsx

```diff
@@ -129,6 +217,15 @@ function OnboardingForm({  }: Props): ReactElement {
           </Step>
         ))}
       </Stepper>
+
```

Comment:
If everything is working fine we can remove this commented code. Or if you want this for reference don't comment it.

Reply:
I think I need this in case we are using custom icons for stepper in future.

Reply:
okay then.